docs/docs/guides/tracking/tracing.mdx

```diff
@@ -79,6 +79,29 @@ def my_function(name: str):
 print(my_function.call("World"))
 ```
 
+:::note
+If your op is a method on a class, you need to pass the instance as the first argument to the op (see example below).
+:::
+
+
+```python showLineNumbers
+import weave
+
+# Initialize Weave Tracing
+weave.init("intro-example")
+
+class MyClass:
+    # Decorate your method
+    @weave.op
+    def my_method(self, name: str):
+        return f"Hello, {name}!"
+
+instance = MyClass()
+
+# Call your method -- Weave will automatically track inputs and outputs
+instance.my_method.call(instance, "World")
```

Comment:
What is the error message that occurs when this is not the case - can we give a good instructional error?

Reply:
It's the default `ValueError: missing arg ____`, but we can make it extra explicit since this is a special func

Reply:
You get an `OpCallError` with clear instructions to pass self:
![image](https://github.com/user-attachments/assets/63fe4d76-429a-457b-ad8f-ed87e33c017a)